fiat_p256.opam

```diff
@@ -12,14 +12,16 @@ build: [
  ["dune" "subst"] {pinned}
  ["dune" "build" "-p" name "-j" jobs]
  ["dune" "runtest" "-p" name] {with-test}
+ ["dune" "build" "@fmt" "-p" name] {with-test}
 ]
 depends: [
   "alcotest" {with-test}
   "asn1-combinators" {with-test}
   "cstruct" {>= "3.5.0"}
-  "dune" {build & >= "1.6.0"}
+  "dune" {build & >= "1.7.0"}
   "hex"
   "hex" {with-test}
+  "ocamlformat" {with-test & = "0.8"}
```

Comment:
this might cause problems until `ocamlformat` can handle several versions at the same time, but there's no better solution for now.

Reply:
BTW can you add `version=0.8` to `.ocamlformat`?

Reply:
I dont think this constraint should be a `with-test` -- it will just cause unnecessary problems with opam constraints when something else has an ocamlformat constraint.  Perhaps just manually install ocamlformat in the CI?